### PR TITLE
use const / let instead of var

### PR DESCRIPTION
**Pull request by anatoliy8493**

### CODE REVIEW

File: src/_lib/assign/index.js [hunk 1]
```diff
@@ -7,7 +7,7 @@ export default function assign(target, dirtyObject) {
 
   dirtyObject = dirtyObject || {}
 
-  for (var property in dirtyObject) {
+  for (let property in dirtyObject) {
```
Comment (TrySound) [inline src/_lib/assign/index.js]: I think this can be const

Comment (anatoliy8493) [reply src/_lib/assign/index.js]: fixed

**Comment by kossnocorp** [comment]
I appreciate the effort, but there's no way I'm going to go through every changed line, and I can't merge the PR without doing it.

Please tell me what tools you used to transform the code so I can do it myself.

Thanks!

**Comment by anatoliy8493** [comment]
> I appreciate the effort, but there's no way I'm going to go through every changed line, and I can't merge the PR without doing it.
> 
> Please tell me what tools you used to transform the code so I can do it myself.
> 
> Thanks!

I did it manually

**Comment by kossnocorp** [comment]
@anatoliy8493 but how?

**Pull request closed without merging**